### ISSUE TITLE
Validator accepting states with invalid state names exceeding the 80 character limit.

### ISSUE THREAD

**Issue by pwtyler** [ChristopheBougere/asl-validator#69]
**Describe the bug**
State names have an 80-character limit enforced by AWS, but not caught by asl-validator.

On deploy, AWS throws:
```
An error occurred (InvalidDefinition) when calling the CreateStateMachine operation: Invalid State Machine Definition: 'INVALID_STATE_NAME: Invalid State name: State 'This is an exceptionally long state name that I know will fail when I try to deploy to AWS' exceeds the 80-character limit allowed by the service at /States/This is an exceptionally long state name that I know will fail when I try to deploy to AWS'
```
**To Reproduce**
Please provide a complete state machine definition:
```json
{
	"Comment": "Passing asl-validator but fails to deploy.",
	"StartAt": "Hello",
	"States": {
		"Hello": {
			"Type": "Pass",
			"Next": "This is an exceptionally long state name that I know will fail when I try to deploy to AWS"
		},
		"This is an exceptionally long state name that I know will fail when I try to deploy to AWS": {
			"Type": "Pass",
			"End": true
		}
	}
}
```

**Expected behavior**
asl-validator would catch the invalid state name and raise an error.

**Version:**
1.9.5

**Comment by ChristopheBougere** [comment]
Thanks for reporting @pwtyler 
That's fixed in 1.9.7
I wonder why [max length was set to 128 chars initially](https://github.com/ChristopheBougere/asl-validator/pull/71/commits/31be1399b662bba25a089f5b089d8aff2f41bd35#diff-301118273effc3b32be9ba1155e57b3e68d89ec588717a37c3036c38df4d60f9L15) 🤔

**Comment by massfords** [comment]
@ChristopheBougere I haven't tested on AWS with long names but the Amazon States Language says the state names can be 128 chars. This is probably where you got 128 from. 

> The state name, whose length MUST BE less than or equal to 128 Unicode characters

https://states-language.net/spec.html#states-fields
